roles/openshift_logging_kibana/tasks/main.yaml

```diff
@@ -141,7 +143,9 @@
     dest: "{{ tempdir }}/templates/oauth-client.yml"
   vars:
     kibana_hostname: "{{ openshift_logging_kibana_hostname }}"
+    kibana_ops_hostname: "{{ openshift_logging_kibana_ops_hostname }}"
```

Comment:
@jcantrill 
> This breaks the pattern because..... we iterate over the role multiple times for ops and non-ops scenarios?

Right.  The first iteration creates the kibana-proxy oauthclient with the kibana hostname.  The second iteration replaces the kibana-proxy oauthclient, using the kibana-ops hostname.